apps/cnquery/cmd/builder/builder.go

```diff
@@ -395,6 +395,7 @@ func kubernetesProviderCmd(commonCmdFlags commonFlagsFn, preRun commonPreRunFn,
 		PreRun: func(cmd *cobra.Command, args []string) {
 			preRun(cmd, args)
 			viper.BindPFlag("namespace", cmd.Flags().Lookup("namespace"))
+			viper.BindPFlag("skip-namespaces", cmd.Flags().Lookup("skip-namespaces"))
```

Comment:
I think we should align this to the kubernetes config `namespaceExlude`, so it should be `--namespace-exlude`

Reply:
namespaceExclude or namespaceSExclude?